samples/subsys/shell/sensor_module/README.rst

```diff
@@ -0,0 +1,94 @@
+.. _sensor_shell_sample:
```

Comment:
You're getting a doc build error (this file is not included in any toctree).  Update your PR to fix this by adding a file ``samples/subsys/shell/shell.rst`` that contains:
```
.. _shell-samples:

Shell Samples
#############

.. toctree::
   :maxdepth: 1
   :glob:

   **/*
```
This .rst file is used to organize and pull in sample documentation for this subsystem (just like the other directories in this subsys section).